shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -395,7 +395,7 @@ class JSBackend {
   private def translateClassMember(
       method: MethodDef[Left[Term, Type]]
   )(implicit scope: Scope): JSClassMemberDecl = {
-    val name = JSField.makeIdentifierValid(method.nme.name)
+    val name = JSField.emitValidFieldName(method.nme.name)
```

Comment:
Why is this called here when the other is called in `Codegen`? This should be called in `Codegen` as well. The name of `method return: T` is `return`, not `"return"`...